packages/cli/src/webhooks/webhook-helpers.ts

```diff
@@ -756,7 +757,9 @@ export async function executeWebhook(
 						);
 					}
 
-					throw new InternalServerError(e.message);
+					const internalServerError = new InternalServerError(e.message);
```

Comment:
we really should start adding the original error as `cause` in all instances of `ResponseError`, and then update `beforeSend` to either check `if (error instanceof ResponseError && error.cause instanceof ExecutionCancelledError)` or `if (error instanceof ResponseError && error.cause.level === 'warning')`, to make sure that we don't have to sprinkle checks like these across the codebase.

Reply:
Definitely agree, tracking here: https://linear.app/n8n/issue/PAY-2145